### DISCUSSION TITLE
Unable to bind a Long Form Field

### DISCUSSION THREAD

**Discussion by APEWAW** [Budibase/budibase#12940] [Help]
Hello, 

How to bind a long form field? It is not included in the available bindings options (only Text Field are listed) and by typing {{  }} I get the following error message: "Current Handlebars syntax is invalid, please check the guide here for more details".

Thank you for your help and feedback!

**Comment by linear** [comment]
<p><a href="https://linear.app/budibase/issue/BUDI-7971/unable-to-bind-a-long-form-field">BUDI-7971 Unable to bind a Long Form Field</a></p>
